### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by SlavaJSFE**
### Done:
- [x] Task 3.1 
- [x] Task 3.2
- [x] Task 3.3

### Additional (optional) tasks:
- [x] +1 - ES6 modules are used for Product Service implementation
- [x] +1 (All languages) - Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
- [x] +1 (All languages) - Main error scenarios are handled by API ("Product not found" error).

### CODE REVIEW

File: product-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+  },
+  functions: { hello, getProductsList, getProductsById },
```
Comment (abachyla) [inline product-service/serverless.ts:25]: Is `hello` function needed? Please remove it if it's not going to be used

**Review by abachyla — APPROVED** [review]

File: product-service/serverless.ts (hunk 1 above)
Comment (SlavaJSFE) [reply product-service/serverless.ts:25]: `hello` function don't interrupt the rest functionality. I need it for test proposes.

**Comment by abachyla** [comment]
Score: 7/11

* [x] +1 Product Service Serverless config contains configuration for 2 lambda functions, API is not working at all, but YAML configuration is correct
* [x] +1 The getProductsList lambda function returns a correct response
* [x] +1 The getProductsById lambda functions return a correct response code
* [x] +1 Your own Frontend application is integrated with Product Service (/products API) and products from Product Service are represented on Frontend.
* [ ] +1 - Async/await is used in lambda functions
* [x] +1 - ES6 modules are used for Product Service implementation
* [ ] +1 - Webpack is configured for Product Service
* [ ] +1 (All languages) - SWAGGER documentation is created for Product Service
* [ ]  +1 (All languages) - Lambda handlers are covered by basic UNIT tests (NO infrastructure logic is needed to be covered)
* [x] +1 (All languages) - Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
* [x] +1 (All languages) - Main error scenarios are handled by API ("Product not found" error).

File: product-service/serverless.ts (hunk 1 above)
Comment (abachyla) [reply product-service/serverless.ts:25]: no problem, thanks for clarification :)

**Pull request merged**